aria/orchestrator/execution_plugin/ctx_proxy/server.py

```diff
@@ -150,43 +150,101 @@ def __exit__(self, *args, **kwargs):
         self.close()
 
 
-def _process_ctx_request(ctx, args):
+def _process_ctx_request(ctx, args): # pylint: disable=too-many-branches,too-many-statements
     current = ctx
-    num_args = len(args)
     index = 0
+
```

Comment:
This was the whole point of this PR.

The previous implementation assumed either one arg or two: one arg meant "read" and two args meant "write". But this required us to support dot notation so that the one arg could refer to a deep nested attribute. Removing dot notation requires a different to support "write".

This PR also supports working on the return value of function calls.